qemu/tests/nvdimm_mapsync.py

```diff
@@ -47,8 +47,15 @@ def run(test, params, env):
             vm.wait_for_login()
             vm_pid = vm.get_pid()
 
-            output = process.system_output(check_cmd % vm_pid).decode()
-            if 'sf' not in re.findall('(?:VmFlags: )(.*)', output)[0].split():
+            lines_count = params.get_numeric("lines_count", 100)
+            vmflags = ""
+            while not vmflags and lines_count < 10000:
+                output = process.getoutput(check_cmd % (lines_count, vm_pid))
+                vmflags = re.findall('(?:VmFlags: )(.*)', output)
+                lines_count *= 2
```

Comment:
```
           with open('/proc/%s/smaps' %vm_pid, 'r') as fd:
                content = fd.read()
            vmflags_match = re.search('^[0-9a-fA-F]+-[0-9a-fA-F]+.*nvdimm0(?:.|\n)*?^VmFlags: (.*)\n', content, re.M)
            if vmflags_match:
                vmflags = vmflags_match.group(1).strip()
            else:
                 test.error("Didn't find VmFlags in smaps file")
```

Reply:
Thanks for the code. It works well. Pls help review again. 
 (1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_mapsync: PASS (40.95 s)